mscore/fotomode.cpp

```diff
@@ -404,7 +402,7 @@ void ScoreView::fotoModeCopy()
       {
       // oowriter wants transparent==false
       bool transparent = false; // preferences.pngTransparent;
```

Comment:
If and when #3397 gets merged, this nine needs mending too

Reply:
And that just happened...

Reply:
Will fix